litigious-liberators/onboarding/onboarding.py

```diff
@@ -0,0 +1,136 @@
+from kivy.app import App
+from kivy.uix.label import Label
+from kivy.uix.floatlayout import FloatLayout
+from kivy.uix.screenmanager import ScreenManager, Screen
+from kivy.animation import Animation
+from kivy.clock import Clock
+from functools import partial
+
+lorem = ""
+sentences = [
```

Comment:
Strings in this list are unclear and it is hard to understand at first glance, maybe should have been put into a text document or something.